_includes/neurips/2019/presenters.neurips.2019.html

```diff
@@ -0,0 +1,146 @@
+<!-- Speakers Grid Section -->
+<section id="speakers" class="bg-light-gray">
+    <div class="container">
+        <div class="row">
+            <div class="col-lg-12 text-center">
+                <h2 class="section-heading">Speakers</h2>
+                <h3 class="section-subheading text-muted"></h3>
+            </div>
+        </div>
+        <div class="grid">
+        {% assign featuredItems = site.tags.oral-neurips2019% | sort: "featuredOrder" %} 
```

Comment:
This page won't load, it stops the entire site from loading locally due to the following error:

_line 11): Cannot sort a null object. included in _pages/neurips_2019/neurips-2019-presenters.html_

To fix this, there needs to be _post objects on the site containing the **tags** oral-neurips2019 with numbered **featuredOrder** to sort by in the order you'd like them to be displayed. Please update the _post objects for this workshop to include this information.